sql_generators/active_users_deletion_requests/templates/mobile_deletion_request_query.sql

```diff
@@ -0,0 +1,236 @@
+WITH baseline AS (
+  SELECT
+    submission_date,
+    joined.normalized_channel,
+    client_id,
+    days_since_seen,
+    days_seen_bits,
+    days_created_profile_bits,
+    durations,
+    joined.normalized_os,
+    joined.normalized_os_version,
+    locale,
+    city,
+    country,
+    app_display_version,
+    device_model,
+    first_seen_date,
+    submission_date = first_seen_date AS is_new_profile,
+    uri_count,
+    is_default_browser,
+    CAST(NULL AS string) AS distribution_id,
+    isp,
+    IF(
+      isp = 'BrowserStack',
+      CONCAT('{{ app_value }}', ' BrowserStack'),
+      '{{ app_value }}'
+    ) AS app_name
+  FROM
+    `{{ project_id }}.{{ app_name }}.clients_last_seen_joined` AS joined
+  INNER JOIN
+    `{{ app_name }}.deletion_request` AS request
+  ON
+    client_info.client_id = client_id
+  WHERE
+    joined.submission_date <= @submission_date
+    AND DATE(request.submission_timestamp)
+    BETWEEN @start_date AND @end_date
```

Comment:
submission date isn't a specified parameter for this query:
```suggestion
    joined.submission_date BETWEEN @start_date AND @end_date
```